tests/Goteo/Core/ModelTest.php

```diff
@@ -133,8 +136,8 @@ public function testCreateTable($mock) {
                   `name` VARCHAR(100),
                   PRIMARY KEY (`id`),
                   UNIQUE INDEX (`uniq`) )";
-        // echo $sql;
-        $query = $mock::query($sql);
+        $mock::query($sql);
+        $this->assertTrue(true);
```

Comment:
and this? is there something that depends on this assertion?

Reply:
Yes, we prevent a warning from PHPUnit. Since I haven't found anything else to assert here, and I wanted to get rid of the warning (from PHPUnit 6 onward, a test must have at least one assert), I dealt with this like that.

It's not the best way to solve it, but it doesn't change the current test at all and there's no warning.

Reply:
super, merging then